apis/server/volume_bridge.go

```diff
@@ -64,6 +64,7 @@ func (s *Server) createVolume(ctx context.Context, rw http.ResponseWriter, req *
 			status[k] = v
 		}
 	}
+	status["size"] = volume.Size()
```

Comment:
If you add this line code, then I suggest that use `status := map[string]interface{}{}` instead in Line 58.
If you do it like this, you can eliminate code 
```
if status == nil {
    status = make(map[string]interface{})
}
``` 

WDYT? @rudyfly

Reply:
You are right.